main.js

```diff
@@ -215,7 +281,7 @@ define(function (require, exports, module) {
 
         $(_command).on("checkedStateChange", onCheckedStateChange);
```

Comment:
I know this code already existed but it tends to be fragile when overriding members of objects created by Brackets, especially if you are overriding a private member...  E.g.  `_command._commandFn` is pretty fragile...

This is definitely another PR... But I thought I'd bring it up now. :)